apps/dash-salesforce-crm/requirements.txt

```diff
@@ -0,0 +1,32 @@
+certifi
```

Comment:
Please lock all of these versions, and make sure that they are all required (I don't think all of them should be). We want to keep this file as short as posisble.